examples/realtime/nextjs-authorization-demo/README.md

```diff
@@ -177,7 +181,9 @@ $$
   BEGIN
     INSERT INTO public.profiles (user_id, email) VALUES (NEW.id, NEW.email); RETURN NEW;
   END;
-$$ LANGUAGE plpgsql;
+$$ LANGUAGE plpgsql
+   SECURITY DEFINER
```

Comment:
this is required since `postgres` user is the one who made this function and this is called by either `postgres` or `supabase_auth_admin`.